src/Symfony/Component/Form/Tests/AbstractBootstrap4HorizontalLayoutTest.php

```diff
@@ -31,15 +31,15 @@ public function testRow()
             '/div
     [
         ./label[@for="name"]
-        [
-            ./span[@class="alert alert-danger d-block"]
-                [./span[@class="d-block"]
+        /following-sibling::div[
+            ./input[@id="name"]
+            /following-sibling::span[@class="alert alert-danger d-block"]
+                [./span[@class="mb-0 d-block"]
```

Comment:
@apfelbox The XPath needs to be fixed here now (the `mb-0` class was removed in #27887 in the meantime).

Reply:
So in fact that's now not a dependency problem of different components anymore, but just a conflict with another PR's changes. :)